cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -1441,3 +1441,206 @@ def test_fixture_with_empty_csv(self):
         outputList = json.loads(outputJSON)
         self.compare_length(outputList, expectedList)
         self.compare_list_of_records(outputList, expectedList)
+
+    def test_fixture_with_helpline_field_removed(self):
```

Comment:
Why is this is its own test? Aren't you going to be removing helpline from all your tests?

Reply:
Yes looking at it now, what I did doesn't make much sense. Ty